onnxruntime/core/providers/vsinpu/patches/mlas_crosscompiling.patch

```diff
@@ -1,34 +1,36 @@
 diff --git a/cmake/onnxruntime_mlas.cmake b/cmake/onnxruntime_mlas.cmake
-index e0ccc504d7..6c5aa6ea53 100644
+index 304aa77f54..2d2289f214 100644
 --- a/cmake/onnxruntime_mlas.cmake
 +++ b/cmake/onnxruntime_mlas.cmake
-@@ -335,7 +335,7 @@ else()
-           ${MLAS_SRC_DIR}/qgemm_kernel_udot.cpp
-           ${MLAS_SRC_DIR}/qgemm_kernel_sdot.cpp
+@@ -354,7 +354,8 @@ else()
          )
+         set_source_files_properties(${MLAS_SRC_DIR}/sqnbitgemm_kernel_neon.cpp
+                                     PROPERTIES COMPILE_FLAGS " -march=armv8.2-a+dotprod")
 -        if (NOT APPLE)
++        # if (NOT APPLE)
 +        if (NOT APPLE AND NOT onnxruntime_USE_VSINPU)
```

Comment:
missed # here?

Reply:
Sorry there exists misunderstanding. It was just a hack before because I directly commented out this line .Now I removed this "#" by directly change the code in origin line